src/containers/contactAdmin/contactAdmin-filter-container.js

```diff
@@ -35,34 +41,44 @@ class ContactAdminFilterWrapper extends Component {
     });
     const queryString = filterHelper.getQueryStringByFilter(filterCopy);
 
-    this.props.history.push(this.props.history.location.pathname + queryString);
+    history.push(history.location.pathname + queryString);
   };
 
-  buildInitialFormValues = () => {
-    const filter = filterHelper.trimUndefinedKeys(
-      this.props.contactAdminList.filter,
-    );
+  const buildInitialFormValues = () => {
+    const filter = filterHelper.trimUndefinedKeys(contactAdminList.filter);
     return { ...filter };
   };
 
-  render() {
-    const initialFormValues = this.buildInitialFormValues();
-    return (
-      <>
-        <ContactAdminFilter
-          onSubmit={this.onSubmit}
-          onReset={this.onReset}
-          form_values={this.props.form_values}
-          initialFormValues={initialFormValues}
-        />
-      </>
-    );
-  }
-}
+  const initialFormValues = buildInitialFormValues();
+  return (
+    <>
```

Comment:
Do you need <>?

Reply:
No, we don't need it, but I fixed es-lint errors only and I didn't check this things. I think, we will delete it when we do refactor this files.

Reply:
I think, you can change it in this PR.

Reply:
Okay, but make it as TODO